src/content/community/conferences.md

```diff
@@ -11,7 +11,7 @@ Do you know of a local React.js conference? Add it here! (Please keep the list c
 ## Upcoming Conferences {/*upcoming-conferences*/}
 
 ### RedwoodJS Conference 2023 {/*redwoodjs-conference-2023*/}
-September 26 - 29, 2023. Grants Pass, Oregon + remote (hybrid event) 
```

Comment:
I agree with @rickhanlonii 
Only modifying the code is what we can discuss whether it is worth it or not. Amending paragraphs like these will lead to many conflicts in the forked repos.

Please do not merge this until you have manually reviewed it and unmodified everything but the code.